package.json

```diff
@@ -40,9 +40,9 @@
     "govuk-frontend": "^3.11.0"
   },
   "devDependencies": {
-    "autoprefixer": "^9.6.1",
     "@babel/preset-env": "^7.12.11",
-    "rollup-plugin-babel": "^4.4.0",
+    "ajv": "^6.12.2",
```

Comment:
ajv (json schema validation to check component config is correct) and glob (read files with a wildcard path) were already indirect dependencies so although they are new here they aren't new to the project (no update package lock needed)